cogeo_mosaic/backends/file.py

```diff
@@ -38,23 +40,27 @@ def point(self, lng: float, lat: float) -> Tuple[str]:
         )
 
     def write(self, gzip=None):
+        """Write mosaicjson document to a file."""
         body = dict(self.mosaic_def)
         if gzip or (gzip is None and self.path.endswith(".gz")):
             body = _compress_gz_json(body)
+        else:
+            body = json.dumps(body).encode("utf-8")
```

Comment:
we always write a binary file

Reply:
Yeah I was wondering if that was an issue. I guess this is equivalent to switching between `open(..., 'w')` and `open(..., 'wb')`